gradle.properties

```
@@ -1,5 +1,5 @@
 GROUP=app.cash.wisp
-VERSION_NAME=1.3.9
+VERSION_NAME=1.3.9-SNAPSHOT
```

This should be 1.3.10-SNAPSHOT:
```suggestion
VERSION_NAME=1.3.10-SNAPSHOT
```